hardware/templates/hardware_admin.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base_tabs.html" %}
+{% load static %}
+{% block extra_head %}
+    <script > window.CSRF_TOKEN = "{{ csrf_token }}"</script>
+
+    <!--hardware/static/admin.js-->
+    <script>
+        document.addEventListener("DOMContentLoaded", ()=>{
```

Comment:
Why doing as a script instead of a normal form?

Reply:
There are changes missing, once I can merge the previous PR to hardware_lab we will see the last version. Anyway, I used an ajax approach because this process involved a lot of fast back-and-forth between the client and the server and I thought it could be inconvenient to be reloading the page every time.